apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -2197,16 +2197,16 @@ export const extensions: Extensions = {
                         smsPublisherDeletionError: {
                             generic: {
                                 message: "Error Occurred",
-                                description: "Error occurred while trying to disable SMS OTP."
+                                description: "Error occurred while trying to remove default choreo SMS OTP provider."
```

Comment:
Let's keep this error message as generic as possible.

```suggestion
                                description: "Error occurred while trying to remove the default SMS OTP provider."
```